index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Sharon Odhiambo</title>
+</head>
+
+<body class="header-background">
+  <header class="nav">
+    <a href="#" id="logo">My logo</a>
+    <nav>
+      <ul class="nav-menu" id="nav-menu">
+        <li class="nav-link"><a href="#works">Portfolio</a></li>
+        <li class="nav-link"><a href="#about-myself">About</a></li>
+        <li class="nav-link"><a href="#form-section">Contact</a></li>
+      </ul>
+      <div class="mobile-navbar" id="mobile-navbar">
+        <div id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul class="mobile-nav-menu" id="mobile-nav-menu">
+          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
+          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
- [ ] As you can see below, the toolbar is a little bit different from the Figma design
Your design 👇 
![Screenshot from 2022-05-27 16-14-44](https://user-images.githubusercontent.com/87965534/170706473-6e78c34b-10f8-4b14-b485-e6babfba6921.png)
Figma design 👇 
![Screenshot from 2022-05-27 16-14-05](https://user-images.githubusercontent.com/87965534/170706395-3d69478a-e065-4b9d-ac17-b4f8e64375a4.png)
Please make sure to put the hamburger button to the left, and to style the logo to match the Figma design.